server/src/main/java/com/objectcomputing/checkins/services/memberprofile/MemberProfileRepository.java

```diff
@@ -1,6 +1,7 @@
 package com.objectcomputing.checkins.services.memberprofile;
```

Comment:
findByWorkEmail is added as art of PR 214 (open). I'll remove my changes and rebase commits if the PR isn't merged tomorrow.